src/lib/ecore_avahi/ecore_avahi.c

```diff
@@ -2,6 +2,10 @@
 # include "config.h"
 #endif
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)